app/src/main/res/values/strings.xml

```diff
@@ -82,6 +82,9 @@
     <string name="disconnect">Disconnect</string>
     <string name="don_t_have_an_account">Don\'t have an account?</string>
     <string name="already_have_an_account">Already have an account?</string>
+    <string name="textview">TextView</string>
```

Comment:
Sorry to bother for that, but I believe this goes to the Debug/temporary section?

Reply:
I think there's been a misunderstanding about @so7fie comment. It is TextView which should be moved to another section, not the "have an account" which should be deleted.